### ISSUE TITLE
-D command line option doesn't override parameters in properties file

### ISSUE THREAD

**Issue by RogueJay** [liquibase/liquibase#1680]
## Environment
**Liquibase Version**: Liquibase Community 4.3.0

**Liquibase Integration & Version**: CLI

**Liquibase Extension(s) & Version**: version 4.3.0

**Database Vendor & Version**: PostgreSQL 11

**Operating System Type & Version**: Windows 10

## Description
If you have a param value in the properties file, using the -D switch on the command line does not override that value

## Steps To Reproduce
Create a changeSet, such as 

```
<changeSet>
	<createTable catalogName="myTable" schemaName="public" tableName="templates" tablespace="${tableSpaceParam}">
		<column name="id" type="UUID">
			<constraints nullable="false" primaryKey="true" primaryKeyName="templates_pkey" />
		</column>
	</createTable>
</changeSet>
```

Have the following entry in the liquibase.properties file
	parameter.tableSpaceParam: pg_default

Use the command line (with appropriate user names etc)
	liquibase update -DtableSpaceParam=ts_myapp

## Actual Behavior
The table is created using the tablespace pg_default, rather than the command line value

## Expected/Desired Behavior
The command line value should be used, creating the table in ts_myapp tablespace

**Comment by molivasdat** [comment]
Hi @RogueJay Thanks for writing up this issue.  We will most likely have to clarify what is meant by a command line parameter.  liquibase --username is considered a command line parameter.  -D is indeed a command line parameter to the jvm that is used to call liquibase but need to look at the code to see where that falls in the priority list of overrides.

**Comment by FBurguer** [comment]
Looks like it got fixed for liquibase 4.17.0.

```
<changeSet id="1-createTable" author="fburguer">
	<createTable catalogName="myTable" schemaName="${param123}" tableName="templates" >
		<column name="id" type="UUID">
			<constraints nullable="false" primaryKey="true" primaryKeyName="templates_pkey" />
		</column>
	</createTable>
</changeSet>
```
I tried executing this changeset on Postgres, added `parameter.param123:public` in my properties file and used `liquibase update  -Dparam123=test`. The table was created on the test schema.
